indexing-service/src/main/java/io/druid/indexing/common/task/AppendTask.java

```diff
@@ -48,18 +49,21 @@
 {
 
   private final IndexSpec indexSpec;
+  private AggregatorFactory[] aggregators;
 
   @JsonCreator
   public AppendTask(
       @JsonProperty("id") String id,
       @JsonProperty("dataSource") String dataSource,
       @JsonProperty("segments") List<DataSegment> segments,
+      @JsonProperty("aggregations") List<AggregatorFactory> aggregators,
```

Comment:
added serde UTs, they work just fine without the getter, It doesn't look like that the getter is necessary.